scripts/sign_verify_file_ssh.sh

```diff
@@ -132,20 +140,16 @@ elif [ "$MODE" == "verify" ]; then
     echo "Verifying the signature against allowed signers..."
 
     # Iterate through each principal in the allowed signers file
-    while IFS= read -r line || [[ -n "$line" ]]; do
-        [[ -z "$line" || "$line" == \#* ]] && continue
-
-        # Extract and process each principal
-        principals=$(echo "$line" | cut -d' ' -f1)
-        IFS=',' read -ra principal_list <<< "$principals"
-
-        for principal in "${principal_list[@]}"; do
-            echo "Checking principal: $principal"
-            if ssh-keygen -Y verify -f "$ALLOWED_SIGNERS_FILE" -n file -I "$principal" -s "$SIG_FILE" < "$FILE_TO_SIGN"; then
-                echo "Signature is valid for principal: $principal"
-                exit 0
-            fi
-        done
+    while read -r principal options key
+    do
+        [[ -z "$principal" || "$principal" == \#* ]] && continue
+
+        namespaces=$(echo "$options" | grep -oP "namespaces=\"\K[^\"]+")
```

Comment:
Are we introducing restrictions on how namespaces are defined here? Are `"` required by the format?

Reply:
Double-quotes around the values of options (such as `namespaces` or `valid-before`) are required.